src/transformers/convert_pegasus_tf_to_pytorch.py

```diff
@@ -0,0 +1,157 @@
+import argparse
```

Comment:
Can't believe I'm the one writing this: Copyright would be nice :-)

Reply:
OK let the record show that before I add all these various comments this pr was +657/-15 :)